run_testlist.py

```diff
@@ -527,22 +541,23 @@
                     'env_vars': {
                         'chunk_size': '1048576',
                         'pool_size': '85G',
-                        'n_file': '200000',
+                        'n_file': '300000',
                         'bytes_read': '0',
                         'bytes_write': '0',
                         'tree_depth': '0',
                         'oclass': 'S1',
+                        'dir_oclass': 'SX',
                         'sw_time': '60'
                     },
                     'enabled': False
                     },
                    {'testcase': 'mdtest_easy_c16_s1',
                     # Number of servers, number of clients, timeout in minutes
                     'testvariants': [
-                        (1, 16, 5), # Use n_file: 50k
-                        (2, 16, 5), # Use n_file: 100k
+                        (1, 16, 5),
+                        (2, 16, 5),
                         (4, 16, 5),
-                        (8, 16, 5),
+                        (8, 16, 5), # sw_time 50
```

Comment:
Generally, I left the configurations as when I last ran them and tried to note discrepancies.